applications/StructuralMechanicsApplication/custom_utilities/tangent_operator_calculator_utility.h

```diff
@@ -60,15 +60,21 @@ class TangentOperatorCalculatorUtility
     /// Pointer definition of TangentOperatorCalculatorUtility
     KRATOS_CLASS_POINTER_DEFINITION(TangentOperatorCalculatorUtility);
 
+    /// Definition of size type
+    typedef std::size_t SizeType;
+
+    /// Definition of index type
+    typedef std::size_t IndexType;
+
     /// Definition of the zero tolerance
     static constexpr double tolerance = std::numeric_limits<double>::epsilon();
 
     // Definition of the perturbation coefficients
-    static constexpr double PerturbationCoefficient1 = 1.0e-5;
-    static constexpr double PerturbationCoefficient2 = 1.0e-10;
+    static constexpr double PerturbationCoefficient1 = 1.0e-4;
```

Comment:
why this was redefined? i think that the previous coefficients were the one that worked correctly with PLCd no?